articles/how-to-collect-sensor-data-on-your-iphone-or-android-device-using-matlab/index.md

```diff
@@ -0,0 +1,131 @@
+### How to collect sensor data on your iPhone or Android device using Matlab
+### Introduction
+Have you ever been in a situation where you are trying to build a project but are still waiting on sensors that haven't even left the warehouse? Well,
+there is a solution to this. Did you know you can use your smartphone for those tasks? Smartphones have several sensors, and with the Matlab mobile app,
+you can use some of those sensors to collect and analyze data.
+Sensors are electrical devices that are for the detection of various conditions or objects. They use signals to detect the various conditions and objects.
+Therefore, sensors form an essential part of sensor involved project. This tutorial will look at how we can collect data using smartphones' sensors and use 
+these data for various projects.
+
+### Prerequisites
+- [Matlab](https://www.mathworks.com/products/get-matlab.html?s_tid=gn_getml) installed.
+- Proper understanding of [matlab](https://www.section.io/engineering-education/getting-started-with-matlab/) basics.
+- Mobile matlab installed on your phone.
+
+To install Matlab mobile app on your phone, go to the play store or AppStore and search `Matlab`. Once you locate this, click on the install, and you
+shall have installed Matlab into your phone.
+
+### How to use mobile phone's Matlab to take photos of images
+You can access the camera to take an image or video manually. This image or video you can later use in your program for analysis. To access the camera, 
+- Open the Matlab on your mobile phone.
+> Note that mobile Matlab works online, and you must have an internet connection to use it.
+- Click on the three dashes on the top-left of the Matlab window.
+![location of the dashes](/howto-collect-sensor-data-on-your-iphone-or-android-device-using-matlab/engineering-education/sensor_one.jpeg)
+- Click on the file section.
+![location of the files](/howto-collect-sensor-data-on-your-iphone-or-android-device-using-matlab/engineering-education/sensor_two.jpeg)
+- Click on the `+` sign at the top-right corner.
+- Click on the take a photo to take a photo of your image.
+![tab for taking the photo](/howto-collect-sensor-data-on-your-iphone-or-android-device-using-matlab/engineering-education/sensor_three.jpeg)
+
+Using the camera on the Matlab mobile app is easy and intuitive, just like you would on most applications. After taking an image, it is uploaded to a 
```

Comment:
```suggestion
Using the camera on the Matlab mobile app is easy and intuitive, just like  on most applications. After taking an image, it is uploaded to a Matlab drive to sync all your files and data. 
The image is uploaded in the `.jpeg` format with an autogenerated name. Since you are going to use this image, you can rename it to be something meaningful.

To rename the image, long-press it, click on the `rename` tab, give it a name you wish then click `ok` to save. 
```